### PR TITLE
Alerts user if the client cannot connect to airsim 

### PR DESCRIPTION
**Pull request by brandonspangler1**
Prompts user to follow instructions upon failure to initialize.

### CODE REVIEW

File: Application/LaunchScreen.py [hunk 1]
```diff
@@ -48,13 +50,19 @@ def navToVQAScreen(self):
         # Start Camera
         self.VQAScreen.setupCamera()
 
+    def setText(self):
```
Comment (jenniferolenchak) [inline Application/LaunchScreen.py]: Needs a more specific function name

File: Application/LaunchScreen.py [hunk 2]
```diff
@@ -48,13 +50,19 @@ def navToVQAScreen(self):
...
+        self.ui.button_InitializeClient.setText("Failed to Start. Follow the above instructions.")
 
     def startVQA(self):
         '''Initialize AirSim client, setup camera feed, and change window display to VQA Interaction Window'''
+        # Logic to verify the success of initializeAirSimClient() and setupCamera() before switching to the VQA screen
         self.ui.button_InitializeClient.setText("Initializing Client...")
```
Comment (jenniferolenchak) [inline Application/LaunchScreen.py:55]: The text does not switch to "Initializing client" until after it has already failed to connect. We may need to use the same app refresh function from the loading screen here so it is forced to updated.

File: Application/LaunchScreen.py [hunk 3]
```diff
@@ -48,13 +50,19 @@ def navToVQAScreen(self):
         # Start Camera
         self.VQAScreen.setupCamera()
 
+    def setText(self):
+        self.ui.button_InitializeClient.setText("Failed to Start. Follow the above instructions.")
```
Comment (jenniferolenchak) [inline Application/LaunchScreen.py]: When the text of the button was changed, the text extended beyond the button's size and was unreadable.

The text of the button itself should change to be something like "Retry Client Initialization" and a separate pop-up box and/or label should appear specifying that there was an error

**Review by jenniferolenchak — CHANGES_REQUESTED** [review]

File: Application/LaunchScreen.py (hunk 1 above)
Comment (brandonspangler1) [reply Application/LaunchScreen.py]: Refactored to remove function.

File: Application/LaunchScreen.py (hunk 2 above)
Comment (brandonspangler1) [reply Application/LaunchScreen.py:55]: It is switching properly on my machine. I added in the forced refresh, see if that works.

File: Application/LaunchScreen.py (hunk 3 above)
Comment (brandonspangler1) [reply Application/LaunchScreen.py]: Added popup window alerting user of error and set the button text to 'Retry Client Initialization.' Provide feedback on error message.

File: Application/LaunchScreen.py (hunk 2 above)
Comment (jenniferolenchak) [reply Application/LaunchScreen.py:55]: It works now!

File: Application/LaunchScreen.py (hunk 3 above)
Comment (jenniferolenchak) [reply Application/LaunchScreen.py]: image

1. Format the text/window to be in the center. Either left-align the text or manually set the window size to have the error message seemingly in the center [see above for issue]
2. Set the window title to be "Error" so it does not default to saying "python3"
3. Set the error window icon to match our main window icon
4. For the error message itself, can we expand the user directions? Maybe something like:
`"Error initializing the client. Before initializing the client, please ensure that you have launched the AirSim Unreal Engine Environment and that the quadrotor drone is loaded completely. Select "Retry Client Initialization" to try again.`

For the main window, items 2 and 3 are done as such:
    `stackedWidget.setWindowTitle("DroneVQA")`
    `stackedWidget.setWindowIcon(QIcon("Images/Logos/logo_drone_only.png"))`

and the result is that the top of the main window appears as such:
image

Comment (jenniferolenchak) [reply Application/LaunchScreen.py]: @brandonspangler2

Comment (brandonspangler1) [reply Application/LaunchScreen.py]: I believe they are all fixed. The popup looks different on Mac so cannot be 100% sure.

**Review by jenniferolenchak — APPROVED** [review]

**Pull request merged**